data_visualization_in_open_source_software/data_visualization_in_open_source_software.md

```diff
@@ -232,23 +228,25 @@ Correct answers include color, position on axis/dimension, shape, line type, siz
 </div>
 ****
 
-Which of the following statements is the **most** true of a typical workflow for building new data visualizations?
+For someone who has spent a lot of time getting good at `ggplot2` or `seaborn`, which of the following would likely be part of their typical workflow when building a new data visualization? Select all that apply.
 
-[( )] People new to data visualization should start by copying code from example plots, but as they get better they'll be able to write plot code themselves
-[( )] It's best to plan your plot carefully from the beginning so that you can get it right the first time and don't have to re-work it
-[(X)] You should pick a plot type that is a good fit both for the types of variables you have and what idea you want your plot to communicate
-[[?]] Hint: Although all of these answers have some element of truth to them, only one is **completely** true.
-[[?]] Hint: See [data visualization workflow](#data-visualization-workflow) for a review.
+[[X]] Spending time thinking about (and maybe drawing) the plot they want before they start coding
+[[X]] Searching for example code for similar plots online
+[[X]] Going through several different versions of their code before they get a plot that they want to keep
 ****
 <div class = "answer">
-Although the first two answers seem plausible, they don't really describe a typical workflow:
+All of these are typical for experienced programmers!
+
+Sometimes it can feel discouraging when, as a beginner, you feel like everything you code takes several false starts before you make any real progress. The truth is that writing code is full of false starts, even for the most expert programmers. Everyone, including experts, reuses code from example plots and previous visualizations (that's why [galleries of example plots](#data-visualization-workflow) are so popular!). You shouldn't expect to be able to write `ggplot2` or `seaborn` code from a blank slate. As with so many programming skills, the main difference between a `ggplot2`/`seaborn` expert and a novice is that the expert knows exactly what to google!
```

Comment:
I think the reworking of this question makes a lot more sense, and I super love how it also turned into an opportunity for normalization of the sometimes frustrating and iterative process that is learning how to code :)